data/hai/unfettered 0 prologue.txt

```diff
@@ -515,3 +518,1020 @@ mission "Unfettered Jump Drive Trading"
+			`	As he passes near you he quickly speaks, "It'll be a few weeks before I'm ready for more action."`
+				accept
+			label stay
+			`	You stay there, surely impressing all Hai present! However it also means you get hit by the tail of the monster as it jumps out to hit you, and everything gets dark.`
+			label poisoned
+			`	You hear voices around you, someone screaming for help, but also your recent opponent speaking calmly. "There is no cure for humans," you hear as you sink deeper and deeper, down to the edge of unconsciousness, "there is nothing to be done. This is a fight they will have to win on their own."`
+			`	Suddenly the world darkens, and you fall. When you open your eyes again, all is vivid, as if you had never seen before. You see the scorpion-like creature that attacked you, clearer than ever, in all its might and with a deep spark of intelligence in its eyes, before it swiftly moves away.`
+			`	Then you are taken far away by a beam of pure light. You see collapsed civilisations, fighting to reclaim their forgotten glory as they struggle to feed their own people. You see a planet, once beautiful, now being bombarded until not a single trace of life is left. You see stars, seemingly eternal, blasted away with all surrounding space. You see automated ships, repairing wounds.`
+			action
+				set "unfettered: heshi trip"
+				"unfettered rep: Si-Ra" ++
+			`	Except you never wake up. The final words you hear before all is truly dark speak of an ongoing medical crisis taking priority.`
+				to display
+					not "Unfettered Small Medicals Rush: done"
+					not "Unfettered Big Medicals Rush: done"
+					not "<uhai> 1: Bring Medicine: done"
+				die
+			`	Until, finally you are brought back by that same beam of light, and it is all you can see.`
+			choice
+				`	(Wake up.)`
+				`	(Give up.)`
+					die
+			`	You wake up in a hospital bed, the Captain is there and speaks very quietly, almost gently, "That was maybe not smart, lad, but it sure was brave! You have what it takes to be one of us, just make sure you don't get killed next time." He then moves to the door behind you.`
+				goto end
+			label run
+			action
+				log "Defeated an Unfettered Captain in a duel, but it seems that was not the whole test. The second part involved a monstrous scorpion from which it felt like a good idea to run at the time. The Unfettered did not see it that way, however."
+			`	You start running away and hear the Captain speak to his crew, loud enough for you to hear, "The monkey is running away from danger. It would not be fit to join us indeed. It will not fight for us."`
+				decline
+			label shoot
+			`	You get your weapon out and shoot the monster as it attempts to rush out of its aquarium. You are not sure if your shot over the aquarium hit it, because of the erupting water that came out your way as the beast emerged, but at least you are still alive!`
```

Comment:
```suggestion
			`	You get your weapon out and shoot the monster as it attempts to rush out of it's aquarium. You are not sure if your shot went over the aquarium or hit it, because of all the erupting water that came out as the beast emerged, but at least you are still alive!`
```